.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+# Note: don't use this config for your own repositories. Instead, see
+# "Version control integration" in README.md.
+exclude: ^(fixtures/)
+repos:
+-   repo: local
+    hooks:
+    - id: black
```

Comment:
I would prefer that pre-commit uses its own version of tools an not the local ones like this for black:
```
- repo: https://github.com/psf/black
  rev: stable
  hooks:
    - id: black
      language_version: python3.6
```

Reply:
How about using the version from within the virtual environment? I'd like that better than installing yet another version of a tool alongside.